src/txs/swap/TFMSwapForm.tsx

```diff
@@ -24,7 +24,7 @@ import { Checkbox } from "components/form"
 import { Read } from "components/token"
 
 /* tx modules */
-import { getPlaceholder, toInput } from "../utils"
+import { getPlaceholder, toInput, CoinInput } from "../utils"
```

Comment:
Please revert this since TFM Swap only support Terra 2.0.

Reply:
Reverted all changes to this form.